docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     ipc: host
     network_mode: host
     privileged: true
-    command: npm run test:e2e
+    command: bash -c 'npm run setup:e2e && npm run test:e2e'
```

Comment:
We can run setup as part of `test:e2e`, this ensures that it works when ran locally as well.